templates/deployment-job.yml

```diff
@@ -9,19 +9,18 @@ jobs:
 - job: validateParameters
   displayName: 'Validate template parameters'
   steps:
-    - bash: $(System.DefaultWorkingDirectory)/scripts/validate-template-parameters.sh
+    - bash: $(System.DefaultWorkingDirectory)/scripts/validate-template-parameters.sh '$(REVIEW_APP_NAME_ROOT)$(System.PullRequest.PullRequestId)'
```

Comment:
This bash script doesn't take an argument

Reply:
Correct, removed the parameter and probably can either remove or refactor this task to check all the secrets and variables fetched from keyvault. (Probably task for later on).